pkg/visor/init.go

```diff
@@ -180,37 +181,34 @@ func initDiscovery(ctx context.Context, v *Visor, log *logging.Logger) error {
 	return nil
 }
 
-func initSNet(ctx context.Context, v *Visor, log *logging.Logger) error {
-	nc := snet.NetworkConfigs{
-		Dmsg: v.conf.Dmsg,
-		STCP: v.conf.STCP,
-	}
-
-	conf := snet.Config{
-		PubKey:         v.conf.PK,
-		SecKey:         v.conf.SK,
-		ARClient:       v.arClient,
-		NetworkConfigs: nc,
-	}
-
-	n, err := snet.New(conf, v.ebc, v.MasterLogger())
-	if err != nil {
-		return err
+func initDmsg(ctx context.Context, v *Visor, log *logging.Logger) error {
+	if v.conf.Dmsg == nil {
+		// todo: this is preserved behavior from snet. Do we need it?
```

Comment:
We should just stop execution here. Without dmsg configured there is virtually nothing a visor can be used for.